soc/arm64/nxp_imx/imx8m/linker.ld

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2020 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <autoconf.h>
+
+#if defined(CONFIG_SOC_IMX8MM_A53)
+#include <linker_a53.ld>
+#endif
```

Comment:
this looks suspicious. If `!CONFIG_SOC_IMX8MM_A53` you get an empty linker script?

Reply:
i.MX8M includes: i.MX8MQ/M/N/P. The patchset currently only add i.MX8MM, so I add this condition check. When we add more SoC support, this condition check could be removed or extended.

Reply:
ok